modules/dummy-module-abc.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+// Epic CNF-792 (4.8)
+
+:_content-type: PROCEDURE
+[id="gathering-data-about-your-cluster-using-must-gather_{context}"]
+= Gathering data about your cluster using the must-gather command
+
+The Performance Profile Creator (PPC) tool requires `must-gather` data. As a cluster administrator, run the must-gather command to capture information about your cluster.
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+* The OpenShift CLI installed.
+
+.Procedure
+
+. Optional: Verify that a matching machine config pool exists with a label:
++
+[source,terminal]
+----
+oc describe mcp/worker-rt
```

Comment:
```suggestion
$ oc describe mcp/worker-rt
```

Reply:
At the start of the command, the `$` sign must be used if a command is being run by a user instead of a root.

https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output